elementary/monitor/fetchers/alerts/alert_filters.py

```diff
@@ -18,24 +18,30 @@ def filter_alerts(
         List[SourceFreshnessAlert],
         List[MalformedAlert],
     ],
-    filter: Optional[SelectorFilterSchema] = None,
+    filter: SelectorFilterSchema = SelectorFilterSchema(),
 ) -> Union[
     List[TestAlert],
     List[ModelAlert],
     List[SourceFreshnessAlert],
     List[MalformedAlert],
 ]:
-    if filter is None:
-        return alerts
     filtered_alerts = []
-    if filter.tag:
+    # If the filter is empty, we want to return all of the alerts
+    if not filter.selector:
+        filtered_alerts = alerts
+    elif filter.tag:
         filtered_alerts = _filter_alerts_by_tag(alerts, filter)
     elif filter.model:
         filtered_alerts = _filter_alerts_by_model(alerts, filter)
     elif filter.owner:
         filtered_alerts = _filter_alerts_by_owner(alerts, filter)
     elif filter.node_names is not None:
         filtered_alerts = _filter_alerts_by_node_names(alerts, filter)
+    # If the filter contains a filter that we don't support, we want to return an empty list of alerts
+    elif filter.selector:
+        logger.error(
+            f"An unsupported alerts selector has been provided - {filter.selector}!\nNo alert has been sent!"
+        )
```

Comment:
I think desired behaviour in case of an invalid selector is "don't filter and just send all alerts forward", not "filter everything out"